src/screens/DomainScreen/elements/Header/elements/ActionButtonGroup.js

```diff
@@ -148,7 +148,7 @@ const styles = StyleSheet.create({
   },
   unblockButtonText: {
     fontSize: normalizeFontSize(12),
-    color: COLORS.white,
+    color: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Changing the color from `COLORS.white` to `COLORS.almostBlack` for the unblockButtonText might affect readability if the background color is dark. Please ensure that this color change maintains sufficient contrast for accessibility.

```diff
-    color: COLORS.white
+    color: COLORS.almostBlack
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->